tools/autograd/derivatives.yaml

```diff
@@ -574,6 +591,7 @@
 - name: ger(Tensor self, Tensor vec2) -> Tensor
   self: grad.mv(vec2.conj())
   vec2: grad.t().mv(self.conj())
+  result: at::ger(self_t, vec2_p) + at::ger(self_p, vec2_t)
```

Comment:
I don't see a test for ger, should it go here? https://github.com/pytorch/pytorch/blob/d1638820935455b4c7d2c6c60baf46827a1bc9a8/torch/testing/_internal/common_methods_invocations.py#L5605-L5609

Reply:
outer actually isn't an alias at all...
It is ger that calls into outer. But weirdly enough, ger has a backward formula while outer is a composite op.

@mruberry do you know why these two ops that are supposed to be alias don't have the same backward definition (one has an explicit formula and the other is composite)?

Reply:
That is weird! Almost definitely an oversight. I would expect `ger` to be a proper alias for `outer`.

Reply:
Ok, sending a PR to fix this :)